### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EduPonz**
This PR takes the work done in #503 and applies some requested changes

### CODE REVIEW

**Comment by EduPonz** [comment]
DCO fails due to difference between signer and committer. @hidmic should I signed the commits?

**Comment by hidmic** [comment]
@EduPonz if I'm not mistaken, the problem is the discrepancy between the previous author and his previous sign. You may sign them as well, but I won't solve the DCO check failures.

File: README.md [hunk 1]
```diff
@@ -60,27 +60,50 @@ If `RMW_FASTRTPS_PUBLICATION_MODE` is not set, then both `rmw_fastrtps_cpp` and
 
 ### Full QoS configuration
 
-With `rmw_fastrtps`, it is possible to fully configure Fast DDS using an XML file as described in [Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/xml_configuration.html).
-When configuring the middleware using XML files, there are certain points that have to be taken into account:
+Fast DDS QoS policies can be fully configured through a combination of the [rmw QoS profile] API, and the [Fast DDS XML] file's QoS elements. Configuration depends on the environment variable `RMW_FASTRTPS_USE_QOS_FROM_XML`.
 
 1. ROS 2 QoS contained in [`rmw_qos_profile_t`](http://docs.ros2.org/latest/api/rmw/structrmw__qos__profile__t.html) are always honored, unless set to `*_SYSTEM_DEFAULT`.
 In that case, XML values, or Fast DDS default values in the absences of XML ones, are applied.
```
Comment (hidmic) [inline README.md:66]: @EduPonz nit^2: 
```suggestion
In that case, XML values, or Fast DDS default values in the absence of XML ones, are applied.
```

**Review by hidmic — APPROVED** [review]
LGTM pending DCO resolution

File: README.md (hunk 1 above)
Comment (EduPonz) [reply README.md]: Done in 6209d7d

**Comment by EduPonz** [comment]
> @EduPonz if I'm not mistaken, the problem is the discrepancy between the previous author and his previous sign. You may sign them as well, but I won't solve the DCO check failures.

I think you're right. I've signed the changes just in case anyways. I don't what else can be done other than remove the original author's and make their changes as if I did them. What's your suggestion?

**Comment by EduPonz** [comment]
I stand corrected, signing them all myself did the trick!

**Comment by EduPonz** [comment]
Hi @hidmic ,

Do you want me to do something else here?

**Comment by clalancette** [comment]
I'm going to go ahead and merge this since hidmic is unavailable this week.  Also, since it is a change to the README.md any follow-ups should be easy.  Thanks for the contribution!

**Comment by hidmic** [comment]
Thanks for taking care @clalancette

**Pull request merged**